### PR TITLE
Walk back to the first Pokestop encountered

### PR DESCRIPTION
**Pull request by pashorizer**
Added option to walk back to starting pokestop after certain timelimit

 Set Timer in seconds to go back to the starting PokeStop (-1 disabled)
timerWalkToStartPokeStop= 600 // e.g. 10 minutes

### CODE REVIEW

File: src/main/kotlin/ink/abb/pogo/scraper/tasks/Walk.kt [hunk 1]
```diff
@@ -15,6 +15,8 @@ import ink.abb.pogo.scraper.Context
 import ink.abb.pogo.scraper.Settings
 import ink.abb.pogo.scraper.Task
 import ink.abb.pogo.scraper.util.Log
+import ink.abb.pogo.scraper.util.inventory.hasPokeballs
+import ink.abb.pogo.scraper.util.map.canLoot;
```
Comment (pr0ves) [inline src/main/kotlin/ink/abb/pogo/scraper/tasks/Walk.kt]: There's already this import at L21.

**Comment by pashorizer** [comment]
@pr0ves fixed

**Comment by pr0ves** [comment]
Is there a way to implement walkRoute()? Because right now when your walkBack timer ends it will walk straight to the first pokestop, right?

**Comment by pashorizer** [comment]
Now it works for walk on streets aswell

File: src/main/kotlin/ink/abb/pogo/scraper/Bot.kt [hunk 2]
```diff
@@ -125,6 +133,16 @@ class Bot(val api: PokemonGo, val settings: Settings) {
             WebServer().start(settings.guiPort, settings.guiPortSocket)
             ctx.server.start(ctx, settings.guiPortSocket)
         }
+
+
+        if (settings.timerWalkToStartPokeStop != -1L)
```
Comment (pr0ves) [inline src/main/kotlin/ink/abb/pogo/scraper/Bot.kt]: Here it's better if you put `settings.timerWalkToStartPokeStop<=0`.
What if someone puts -2 as timerWalkToStartPokeStop?

**Comment by pashorizer** [comment]
@pr0ves up

**Pull request merged**